Assets/SEE/XR/RadialSelection.cs

```diff
@@ -0,0 +1,409 @@
+using SEE.Controls.Actions;
+using SEE.GO;
+using System.Collections.Generic;
+using System.Linq;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.InputSystem;
+using UnityEngine.UI;
+using UnityEngine.XR.Interaction.Toolkit;
+using SEE.Utils;
+
+/// <summary>
+/// This class is used, to open a
+/// radial menu in VR.
+/// This script is based on this tutorial: https://www.youtube.com/watch?v=n-xPN1v3dvA
+/// </summary>
+public class RadialSelection : MonoBehaviour
+{
+    /// <summary>
+    /// The number of radial-parts.
+    /// </summary>
+    int numberOfRadialPart = 0;
+    /// <summary>
+    /// The prefab for the single radial-parts.
+    /// </summary>
+    public GameObject radialPartPrefab;
+    /// <summary>
+    /// The canvas on which the radial should be shown.
+    /// </summary>
+    public Transform radialPartCanvas;
+    /// <summary>
+    /// The angle between the radial-parts.
+    /// </summary>
+    public float angleBetweenPart = 10;
+    /// <summary>
+    /// The radial-parts which got spawned.
+    /// </summary>
+    private List<GameObject> spawnedParts = new List<GameObject>();
```

Comment:
We can use the new-style new here:
```suggestion
    private List<GameObject> spawnedParts = new();
```